src/main/scala/io/iohk/ethereum/consensus/ethash/EthashBlockCreator.scala

```diff
@@ -28,20 +28,24 @@ class EthashBlockCreator(
   lazy val blockchainConfig = consensus.blockchainConfig
 
   def getBlockForMining(
-      parentBlock: Block,
+      parentBlock: Option[Block],
```

Comment:
I don't think it makes sense to call this method with optional parentBlock as parentBlock is essential for execution. We could return an error one level higher, from the place this method is called